### PR TITLE
loading improvements

### LINKED ISSUE #765: let's use an animated loader so everyone knows we care 😍
we tend to default to `loading...` but a small loading image or animated html element will give our app a nice sense of aliveness 🚤

### PR DESCRIPTION
**Pull request by jolesbi**
- changed loading icon
- added loading icon to some missing spots
- removed content from initial html
- set background color for electron

loading-screenshot

closes #765

### CODE REVIEW

File: app/index.ejs [hunk 1]
```diff
@@ -1,25 +1,19 @@
...
+    </script>
+  <% } %>
 
-<style>
-<%= htmlWebpackPlugin.options.styles %>
-</style>
+  <body style="background: #15182d">
+    <div id="app" style="background: #15182d">
+      <span>Loading...</span>
+      <!-- <img src="imgs/loader.svg" alt="spinner"> -->
```
Comment (jolesbi) [inline app/index.ejs]: does anyone know how to get an image to load here?

**Comment by faboweb** [comment]
Can we get some screenshots of the visual changes?

File: app/index.ejs (hunk 1 above)
Comment (faboweb) [reply app/index.ejs]: What is the problem? Maybe the path "imgs/loader.svg" is not correct in production?

**Comment by jolesbi** [comment]
> Can we get some screenshots of the visual changes?

yes of course! sorry! i took them but forgot to include them! they are now above.

File: app/index.ejs (hunk 1 above)
Comment (jolesbi) [reply app/index.ejs]: yeah i think this file ends up in the dist folder

**Comment by jolesbi** [comment]
ok this is ready to go!

**Comment by faboweb** [comment]
Summing up: This removes the preloader showing before the app is actually loaded. We don't need the preloader anymore, as initialization is done asynchronous via IPC.

**Pull request merged**